.travis.yml

```diff
@@ -19,8 +19,10 @@ env:
     #- DEPS="pytest~=3.1.0"
 
 stages:
+  - name: test
+    if: tag IS NOT present
```

Comment:
Never push a tag without having it tested first.. ;)

Reply:
Definitely